pkg/controller/job/job_controller_test.go

```diff
@@ -4788,57 +4793,79 @@ func TestJobBackoffForOnFailure(t *testing.T) {
 	}{
 		"backoffLimit 0 should have 1 pod active": {
 			1, 1, 0,
-			false, []int32{0}, v1.PodRunning,
+			false,
```

Comment:
Revert this for now.
These are very old test cases, but this is the wrong fix.
It should be:
```golang
{
  parallelism: 1,
  completions: 1,
  ...
}
```
But that's for another PR.

Reply:
Yeah, this is my default vscode linters...

I'll try and figure out how to revert it.